.env.example

```diff
@@ -4,7 +4,8 @@ PRIVATE_KEY=
 # RPC 
 RPC_URL=
 
-# Threshold network BLS Point G2 public key x and y coordinates (all uints) used to construct the public key as a G2 point
+# Threshold network BLS Point G1 private key and Point G2 public key x and y coordinates (all uints) used to construct the public key as a G2 point
+BLS_PRIVATE_KEY=
```

Comment:
Is this just used for testing? At some point we won’t have the whole key (obviously!)

Reply:
That's right, it's only for testing purposes.